frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/details-card.tsx

```diff
@@ -77,14 +88,13 @@ export const DetailsCard: React.FC<DashboardItemProps> = ({
           isLoading={!ocsLoaded}
           data-test-id="cluster-name"
         >
-          {ocsName}
+          {ocsName}s
         </DetailItem>
         <DetailItem
           title={t('ceph-storage-plugin~Provider')}
-          error={!!infrastructureError || (infrastructure && !infrastructurePlatform)}
-          isLoading={!infrastructureLoaded}
+          isLoading={!secretLoaded && !secretError}
         >
-          {infrastructurePlatform}
+          {cephLink ? <ExternalLink href={cephLink} text={CEPH_BRAND_NAME} /> : CEPH_BRAND_NAME}
```

Comment:
Do we not want to externalise it anymore ?

Reply:
{cephLink ? <ExternalLink href={cephLink} text={CEPH_BRAND_NAME} /> : cephBrandName(t)}

Reply:
Its a product name, we can ignore that.

Reply:
ack !!